lib/ui/theme/theme.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+class AppTheme {
+  const AppTheme._();
+  static ThemeData get darkTheme => ThemeData.from(
+        useMaterial3: true,
+        textTheme: const TextTheme(
+          displayLarge: TextStyle(
+            color: Colors.white,
+            fontSize: 48,
+            fontWeight: FontWeight.w600,
+          ),
+          displayMedium: TextStyle(
+            color: Colors.white,
+            fontSize: 32,
+            fontWeight: FontWeight.w500,
+          ),
+          displaySmall: TextStyle(
+            color: Colors.white,
+            fontSize: 16,
+            fontWeight: FontWeight.w700,
```

Comment:
hey @thisisamank why do we need to specify all these here, the defaults are also looking fine only.

Reply:
I added them for 2 primary reasons:
- a lot of widgets take up their default text styles and we might want to override them at a few places with our own text sizes and colors so there we can use these.
- if we want to support light/dark theme with different colors in the theme.